release.yaml

```diff
@@ -23,6 +23,14 @@ rancher-backup:
 rancher-backup-crd:
 - 1.2.100+up1.2.1
 - 2.1.1+up2.1.1-rc1
+harvester-cloud-provider:
+- 100.0.1+up0.1.8
+- 100.0.0+up0.1.8
+- 100.0.0+up0.1.8
+harvester-csi-driver:
+- 100.0.1+up0.1.9
+- 100.0.0+up0.1.9
+- 100.0.0+up0.1.9
```

Comment:
why are there so many? There should just be `100.0.1+up0.1.9`

Reply:
ya merged bad, ill fix